src/Components/AlbumName/AlbumName.module.css

```diff
@@ -0,0 +1,23 @@
+.title {
+   margin: 0;
+   padding-top: 8px;
+   display: inline-block;
+}
+
+.title a{
+    display: inline-block;
+    font-size: 14px;
+    text-decoration: none;
+    color:  #fff;
+    transition : all .5s ease;
+    font-family:"Helvetica";
+    font-weight: 600;
+
+    
```

Comment:
just some unneeded spacing here

Reply:
Thank you for pointing this out. I think I should be more careful about the cleanness of the code.